examples/lora_dreambooth/convert_peft_sd_lora_to_kohya_ss.py

```diff
@@ -19,7 +19,7 @@
 
 def get_module_kohya_state_dict(module: PeftModel, prefix: str, dtype: torch.dtype) -> Dict[str, torch.Tensor]:
     kohya_ss_state_dict = {}
-    for peft_key, weight in get_peft_model_state_dict(module).items():
+    for peft_key, weight in get_peft_model_state_dict(module, adapter_name=LORA_ADAPTER_NAME).items():
```

Comment:
Here, the user needs to still update the `LORA_ADAPTER_NAME`. Could you please add `adapter_name` as kwarg with default value being `LORA_ADAPTER_NAME` and pass that here.

Reply:
@pacman100 Added adapter_name as kwarg and changed function calls